.mergify.yml

```diff
@@ -0,0 +1,11 @@
+pull_request_rules:
+- actions:
+    merge:
+      method: squash
+  name: Automatically merge pull requests
+  conditions:
+  - status-success=continuous-integration/appveyor/pr
+  - status-success=hydra
+  - label=merge me
+  - ! '#approved-reviews-by>=1'
+  - ! '#changes-requested-reviews-by=0'
```

Comment:
The irony of having `.yml` checked in :laughing:

Reply:
To be fair, I generated it with `dhall-to-yaml` 🙂